src/adstaylorinteg.jl

```diff
@@ -0,0 +1,327 @@
+# This file is part of the TaylorIntegration.jl package; MIT licensed
+
+using TaylorSeries: generate_index_vectors
```

Comment:
I suggest to remove this line; my point is that the actual number of variables and order have been set, so this information has already been calculated and is stored in memory.
```suggestion
```